modules/multi-architecture-scheduling-examples.adoc

```diff
@@ -0,0 +1,20 @@
+// module included in the following assembly 
+//
+//post_installation_configuration/configuring-multi-arch-compute-machines/multi-architecture-compute-managing.adoc
+
+:_content-type: CONCEPT
+[id="multi-architecture-scheduling-examples_{context}"]
+
+= Sample multi-architecture node workload deployments 
+
+There are various cases to consider when you are scheduling workloads on a cluster with compute nodes of different architectures.
+
+//not sure about this format yet
+For example, you can taint a node using a `RuntimeClass` object to avoid unverified workloads, or workloads that are not compatible with an architecture, to be scheduled on that node. 
```

Comment:
maybe we list some of these options as bullet points in more detail?

Reply:
Yeah, I was thinking that as well! I'll add some more of those cases we mentioned. Are there any example yaml that we have for those cases?